notifications/src/main/java/org/opennms/horizon/notifications/kafka/KafkaConsumer.java

```diff
@@ -0,0 +1,29 @@
+package org.opennms.horizon.notifications.kafka;
+
+import org.opennms.horizon.shared.dto.event.AlarmDTO;
+import org.opennms.horizon.shared.dto.event.EventDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class KafkaConsumer {
+    private final Logger log = LoggerFactory.getLogger(KafkaConsumer.class);
```

Comment:
Just for consistency, that's how most of opennms code uses LOG
```suggestion
    private final Logger LOG = LoggerFactory.getLogger(KafkaConsumer.class);
```

Reply:
Do we prefer this line or Lomboks `@Slf4j` annotation?

Reply:
I'm not fan of Lomboks. You can keep this.